kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt

```diff
@@ -0,0 +1,11 @@
+package react.select
+
+import js.objects.jso
+import react.ChildrenBuilder
+import react.ReactDsl
+
+fun <Option : Any, Group : GroupBase<Option>> ChildrenBuilder.CreatableSelect(
```

Comment:
Adapter is redundant with new builder

Reply:
I tried to pass generic type directly to `external val CreatableSelect` but it doesn't work.
How it should looks like?

Reply:
You can find example in `mui-kotlin` playground

Reply:
Done, thanks!